src/main/java/seedu/mindmymoney/command/DeleteCommand.java

```diff
@@ -9,17 +9,20 @@
  */
 public class DeleteCommand extends Command {
     private String input;
+    public Lists itemList;
 
-    public DeleteCommand(String input) {
+
+    public DeleteCommand(String input, Lists listArray) {
```

Comment:
I see this in the other `Command` classes, but I think `listArray` is a redundant name, since `list` and `array` are both words that suggest that the parameter contains some collection. I also think `itemList` isn't that descriptive (what kinds of items does it store?). Consider a name like `expenditureList`?

Reply:
Fixed